superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/transformProps.ts

```diff
@@ -181,6 +181,20 @@ export default function transformProps(chartProps: CccsGridChartProps) {
     JSON: 'jsonValueRenderer',
   };
 
+  const formatIpV4 = (v: any) => {
```

Comment:
doesn't this exist somewhere else too?

Reply:
Yes it does, it lives in the `Ipv4ValueRenderer.tsx` file which exists in the parent directory of this file. However, no matter what I tried, I could not import it and use it (only for this particular function).

Reply:
As per our conversation, this is fine for now and we will clean it up when we revamp the viz before we publish it upstream